src/content-helper/editor-sidebar/excerpt-suggestions/component-panel.tsx

```diff
@@ -331,18 +359,21 @@ const PostExcerptSuggestions = () => {
 						</div>
 					</>
 				) }
-				<Button
-					href="https://docs.parse.ly/plugin-content-helper/#h-excerpt-generator-beta"
-					target="_blank"
-					variant="link"
-				>
-					{ __( 'Learn more about Parse.ly AI', 'wp-parsely' ) }
-					<Icon
-						icon={ external }
-						size={ 18 }
-						className="parsely-external-link-icon"
-					/>
-				</Button>
+
+				{ isDocumentSettingPanel && (
+					<Button
+						href="https://docs.parse.ly/plugin-content-helper/#h-excerpt-generator-beta"
```

Comment:
This link doesn't seem to work, but it's not necessarily our fault. I'll follow up internally about that.

Reply:
Let's use this link: https://docs.wpvip.com/parse-ly/wp-parsely-features/excerpt-generator/